desktop/src/utils/codemod/expressionMethods.js

```diff
@@ -112,8 +127,15 @@ const nodeHasValue = (node, value) => {
  * @return {Collection}      Updated Collection
  */
 export const addFunctionCall = function(object, property, args) {
+  const rangeOfLastMatching = getLastMatchingExpressionRange(this, object, property)
+  const body = _.get(this.nodes(), '[0].program.body')
+  let newFunctionIndex = 0
+  if (rangeOfLastMatching) {
+    newFunctionIndex = _.map(body, 'range').indexOf(rangeOfLastMatching) + 1
+  }
+
   const node = createExpressionStatement(object, property, args)
-  _.get(this.nodes(), '[0].program.body', []).unshift(node)
+  body.splice(newFunctionIndex, 0, node)
```

Comment:
I think it's safer to append function calls to the end of the body rather than put them at the top